labelbox/__init__.py

```diff
@@ -2,6 +2,10 @@
 __version__ = "3.36.1"
 
 from labelbox.client import Client
+from labelbox.data import annotation_types as types
+from labelbox.data import metrics
+from labelbox.data.serialization import NDJsonConverter
+from labelbox.data.serialization import COCOConverter
```

Comment:
Same with these modules. But also no one should ever have to use these directly. Instead I recommend updating `create_from_objects()` to accept annotation types and we handle the conversion ourselves.

I don't see anything wrong with having obscure functions not in the top level lib.

Reply:
Basically follow this pattern that we use for projects
import: https://github.com/Labelbox/labelbox-python/blob/a356b0c993c76678b6b02dadc5289334f74792d3/labelbox/schema/project.py#L40-L43
catch at run-time:
https://github.com/Labelbox/labelbox-python/blob/a356b0c993c76678b6b02dadc5289334f74792d3/labelbox/schema/project.py#L1243-L1248

Reply:
I can remove `COCOConverter` since only a few notebooks use that. @kkim-labelbox is there a ticket to hide NDJsonConverter?

Reply:
We should create a ticker for " we handle the conversion (between AnnotatinoTypes -> ndjson) ourselves." if we haven't/ This is part of annotation v0 scope

Ideally, no need for users to use NDJsonConverter at all

Reply:
It is a very small task. I would say it is part of this work basically.